core/plugins/org.fusesource.ide.camel.validation/src/org/fusesource/ide/camel/validation/model/NumberValidator.java

```diff
@@ -30,22 +31,29 @@
 	public NumberValidator(Parameter parameter) {
 		this.parameter = parameter;
 	}
+	
+	public NumberValidator() {
+	}
 
-	/* (non-Javadoc)
-	 * @see org.eclipse.core.databinding.validation.IValidator#validate(java.lang.Object)
-	 */
 	@Override
 	public IStatus validate(Object value) {
-		// only check non-empty fields
-		if (CamelComponentUtils.isNumberProperty(parameter)) {
-			if (value != null && value.toString().trim().length() > 0) {
-				try {
-					PropertiesUtils.validateDuration(value.toString());
-				} catch (IllegalArgumentException ex) {
+		if (isNonEmpty(value)
+				&& !new CamelPlaceHolderUtil().isPlaceHolder(value)
+				&& (parameter == null || CamelComponentUtils.isNumberProperty(parameter))) {
+			try {
+				PropertiesUtils.validateDuration(value.toString());
+			} catch (IllegalArgumentException ex) {
+				if (parameter != null) {
 					return ValidationStatus.error(NLS.bind(Messages.NumberValidator_messageError, parameter.getName()), ex);
+				} else {
+					return ValidationStatus.error(Messages.numberValidatorMessageErrorWithoutArgument);
 				}
 			}
 		}
 		return ValidationStatus.ok();
 	}
+
+	protected boolean isNonEmpty(Object value) {
```

Comment:
Can we move this method into the org.fusesource.ide.foundation.core.util.Strings class? Strings.isEmpty(String) is very close and this would be a good variation to include in a more centralized utility I think